test/image/mocks/pie_negative_values.json

```diff
@@ -0,0 +1,42 @@
+{
+    "data": [
+      {
+          "labels": [
+            "Apples",
+            "Apples",
+            "Bananas",
+            "Oranges",
+            "Pears",
+            "Pears",
+            "Apples",
+            "Pears"
+          ],
+          "type": "pie",
+          "domain": {"x": [0, 0.4]}
+      },
+      {
+          "labels": [
+              "Apples",
+              "Apples",
+              "Bananas",
+              "Oranges",
+              "Pears",
+              "Pears",
+              "Apples",
+              "Pears"
+          ],
+          "values": [
+              10, -20, 20, 40, 10, -20, 70, 5
+          ],
+          "marker": {"colors": ["", "", "#ccc"]},
+          "type": "pie",
+          "domain": {"x": [0.5, 1]},
+          "textinfo": "label+value+percent"
+      }
+    ],
+    "layout": {
+      "height": 300,
+      "width": 500
+    }
+  }
+  
```

Comment:
Please fix this file. `npm run test-syntax` complains about 
`test/image/mocks/pie_negative_values.json : does not have a trailing new line character`

Reply:
Please also add and commit this image in the `test/image/baselines/` folder.
Thanks!
![pie_negative_values](https://user-images.githubusercontent.com/33888540/145238197-c7c39056-fe88-4d4e-8ac9-10d7de5e9eff.png)

Reply:
Added this image to `test/image/baselines/`